apps/api/src/hyp3_api/api-spec/job_parameters.yml.j2

```
@@ -197,3 +197,8 @@ components:
       description: Include a GeoTIFF in the product package containing displacement values along the Line-Of-Sight (LOS)
       default: false
       type: boolean
+
+    displacement_maps:
+      description: Include GeoTIFFs in the product package containing vertical (VERT) and Line-Of-Sight (LOS) displacements
```

This is the description Heidi drafted for Vertex, for consistency across Vertex/api/sdk:
```suggestion
      description: Include displacement maps (line-of-sight and vertical) in the product package
```